support/certs/tls.go

```diff
@@ -356,11 +369,20 @@ func ReconcileSignedCert(
 		secret.Data[caKey] = append([]byte(nil), ca.Data[opts.CASignerCertMapKey]...)
 	}
 
+	var certValidity = ValidityOneYear
```

Comment:
no need for the `var` keyword here
```suggestion
	certValidity := ValidityOneYear
```